packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.scss

```diff
@@ -48,17 +48,17 @@ cbp-usa-banner {
 
     --cbp-button-padding: var(--cbp-space-1x);
 
-    --cbp-button-color: var(--cbp-color-interactive-secondary-light);
+    --cbp-button-color: var(--cbp-color-interactive-primary-light);
     --cbp-button-color-hover: var(--cbp-color-interactive-secondary-lighter);
-    --cbp-button-color-focus: var(--cbp-color-violet-30);
+    --cbp-button-color-focus: var(--cbp-color-interactive-focus-light);
 
     --cbp-button-color-border-hover: transparent;
     --cbp-button-color-border-focus: transparent;
     --cbp-button-color-border-active: transparent;
 
     --cbp-button-border-radius: 0;
 
-    --cbp-button-color-outline-focus: var(--cbp-color-violet-30);
+    --cbp-button-color-outline-focus: var(--cbp-color-interactive-focus-light);
 
```

Comment:
The active/clicked color needs updating too. Right now, while the mouse is pressed down on the control, the text turns black and essentially disappears. There should be a var for that as well (active I think).

Reply:
updated to override the --cbp-button-color-active token to the focus color to avoid the described behavior